src/elements/earn/pools/poolsTable/PoolsTable.tsx

```diff
@@ -21,9 +21,11 @@ export const PoolsTable = ({ search, setSearch }: Props) => {
   const pools = useAppSelector<Pool[]>((state) => state.pool.pools);
 
   const data = useMemo<Pool[]>(() => {
-    return pools.filter(
-      (p) => p.name && p.name.toLowerCase().includes(search.toLowerCase())
-    );
+    return pools
+      .filter((p) => p.version >= 28)
```

Comment:
is the pool version always comparable to an integer?

Reply:
thats correct